Auth0/WebAuth.swift

```diff
@@ -200,4 +200,20 @@ public protocol WebAuth: Trackable, Loggable {
      - parameter callback: callback called with the result of the WebAuth flow
      */
     func start(_ callback: @escaping (Result<Credentials>) -> Void)
+
+    /**
+     Clear the SSO Cookie in Auth0 and optionally signout from IdP.
+     More info: https://auth0.com/docs/logout
+
+     
+     ```
+     Auth0
+        .webAuth()
+        .clearSession(useFederated: false) { print($0) }
+     ```
+     
+     - parameter federated: Bool to enable signout of the IdP
+     - parameter callback: callback called with the result of the logout
+     */
+    func clearSession(useFederated federated: Bool, callback: @escaping (Bool) -> Void)
```

Comment:
Originally was going to default to `false` (not a fan of optional booleans) then thought the user should understand what method they are calling.